src/nvim/ops.c

```diff
@@ -72,6 +72,12 @@ static yankreg_T y_regs[NUM_REGISTERS];
 static yankreg_T   *y_previous = NULL; /* ptr to last written yankreg */
 
 static bool clipboard_didwarn_unnamed = false;
+
+// for behavior between start_global_changes() and end_global_changes())
+static bool delay_clipboard_update = false;  // clipboard's update needs to be delayed
+static int global_change_count = 0;          // if set, inside global changes
+static bool clipboard_needs_update = true;   // the clipboard was updated
```

Comment:
For our sanity it would help to namespace flags like this. We have `clipboard_didwarn_unnamed` and `clipboard_needs_update`, so `clipboard_delay_update` would remove some entropy.